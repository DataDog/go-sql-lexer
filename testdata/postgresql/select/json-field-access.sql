SELECT data->'customer'->>'name' AS customer_name FROM orders;
