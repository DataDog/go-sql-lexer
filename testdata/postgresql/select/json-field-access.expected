[
    {
        "expected": "SELECT data -> ? ->> ? FROM orders",
        "statement_metadata": {
            "size": 12,
            "tables": [
                "orders"
            ],
            "commands": [
                "SELECT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]